pm_yaml_config/pm_yaml_config.xml

```diff
@@ -0,0 +1,126 @@
+                <param type="text" name="uep_chden" optional="true" label="path to .den_fmt and castep files" help="path to the .den_fmt file containing the electronic density for an Unperturbed Electrostatic Potential optimisation. The corresponding .castep file must be in the same folder and with the same seedname."/>
+                <param type="text" name="castep_command" optional="true" label="castep command" help="command to use to run CASTEP."/>
+                <param type="text" name="castep_param" optional="true" label="PARAM path" help="file path to the CASTEP parameter file."/>
+                <param type="text" name="mu_symbol" optional="true" label="mu symbol" help="the symbol to use for the muon when writing out the castep custom species." />
+                <param type="text" name="dftb_set" optional="true" label="dftb set" help="the parameter set to use for DFTB+. Currently supported are 3ob-3-1 and pbc-0-3. For more information see (the DFTB site)[http://www.dftb.org/parameters/]."/>
+                <param type="text" name="dftb_optionals" value="[]" optional="true" label="dftb optionals" help="additional optional json files to activate for DFTBArgs (for example, dftd3.json will use DFTD3 dispersion forces for 3ob-3-1 if DFTB+ has been compiled to support them)."/>
+                <param type="select" name="dftb_pbc" value="true" optional="true" label="dftp pbc" help="whether to turn on periodic boundary conditions in DFTB+.">
+                    <option value="true">true</option>
+                    <option value="false">false</option>
+                </param>
+            </when>
+        </conditional>
+        <param type="text" name="clustering_method" label="clustering method" value="hier" />
+        <param type="float" name="clustering_hier_t" value="0.2" label="hierarchical clustering t"/>
+        <param type="integer" name="supercell" value="30" label="supercell" multiple="true" help="supercell size and shape to use. This can either be a single int, a list of three integers or a 3x3 matrix of integers. For a single number a diagonal matrix will be generated with the integer repeated on the diagonals. For a list of three numbers a diagonal matrix will be generated where the digonal elements are set to the list. A matrix will be used directly as is. Default is a 3x3 indentity matrix."/>
+        <param type="text" name="k_points_grid" value="[1,1,1]" label="k-points grid" multiple="true" help="list of three integer k-points. Default is [1,1,1]."/>
+        <param type="integer" name="max_scc_steps" value="200" label="max scc steps" help="if applicable, max number of SCC steps to perform before giving up. Default is 200 which is also the default for DFTB+."/>
+    </inputs>
+    <outputs>
+        <data label="configuration" name="out_yaml" format="txt" from_work_dir="output.yaml"/>
```

Comment:
1. This should be able to use the `yaml` (or `yml`?) format now.
2. As for other tools, let's include the `name` parameter in the label.

Reply:
done

Reply:
still needs `yaml` format